integration/exec_test.go

```diff
@@ -12,6 +12,17 @@ import (
 )
 
 func TestExecPS(t *testing.T) {
+	testExecPS(t, false)
+}
+
+func TestUsernsExecPS(t *testing.T) {
+	if _, err := os.Stat("/proc/self/ns/user"); os.IsNotExist(err) {
```

Comment:
Hmm, I don't see that we using such skip strategy for other namespaces, maybe we should just add test mappings and NEWUSER to `newTemplateConfig`?